lte/protos/apn.proto

```diff
@@ -123,12 +123,12 @@ message GatewayAPNConfig {
     APNFilter apn_filter = 2;
```

Comment:
who generates apn_id? how is it used?

What is the purpose of APNFilter?

Reply:
apn_id(Its incorrectly marked as u32, will change this to string) identifies the network wide APN entity. APNFilter is just a future feature, where we can customize the APN specific behavior per gateway on whether we choose to permit or block. I can remove it if you feel its a distraction.